.dotnet/src/Custom/Assistants/CodeInterpreterToolResources.cs

```diff
@@ -12,7 +12,7 @@ public partial class CodeInterpreterToolResources
     public IList<string> FileIds
     {
         get => _fileIds;
-        init
+        set
```

Comment:
Since this is a collection, this one shouldn't have a setter. I wonder if it can just be removed. cc: @trrwilson

Reply:
We can fix removing the setter from the collection in a separate PR. I tried removing it, but since there is an `init` before, a bunch of test cases started failing.